tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -48,8 +48,8 @@ Feature: List a sharing permissions
           },
           "@libre.graph.permissions.roles.allowedValues": {
             "type": "array",
-            "minItems": 4,
-            "maxItems": 4,
+            "minItems": 3,
+            "maxItems": 3,
```

Comment:
@saw-jan right now it gives 4 allowed response. just need to make sure if they are going to change the response according to what is expected otherwise we will endup putting this in expected to failure for the rest.

Reply:
we should not merge PR until the fix is there.